recipes-support/snapd/snapd_2.47.1.bb

```diff
@@ -31,7 +31,8 @@ DEPENDS += "			\
 	glib-2.0		\
 	udev			\
 	xfsprogs		\
-	libseccomp \
+	libseccomp      \
+	libapparmor     \
```

Comment:
AFAICT the libapparmor recipe is gone in the dunfell branch of meta-security, and got renamed to `apparmor`. There's a corresponding `apparmor` distro feature that can be set now too. With this, I think the recipe should conditionally enable apparmor like so:

```
PACKAGECONFIG ??= "${@bb.utils.contains('DISTRO_FEATURES', 'apparmor', 'apprmor', '', d)}"
PACKAGECONFIG[apparmor] = "--enable-apparmor,--disable-apparmor,apparmor,apparmor"
```